backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/PO.java

```diff
@@ -1688,6 +1691,14 @@ protected final void load(final int ID, final String trxName)
 		}
 	}    // load
 
+	private static final PerformanceMonitoringService.Metadata loadMetadata =
+			PerformanceMonitoringService.Metadata
+					.builder()
+					.name("PO")
+					.type(PerformanceMonitoringService.Type.REST_API_PROCESSING)
```

Comment:
not sure but i guess u would want to use some "CRUD" type instead of REST_API_PROCESSING